src/cascadia/ut_app/JsonTests.cpp

```diff
@@ -36,26 +36,27 @@ namespace TerminalAppUnitTests
             return true;
         }
 
-        Json::Value VerifyParseSucceeded(std::string content);

```

Comment:
these changes seem unrelated

Reply:
When I wrote the `KeyBindingsTests::KeybindingsWithoutVkey` and ran it in a debugger I noticed that it created temporary strings, so I improved this code. Would you like me to file this in a separate PR?

Reply:
Meh, no need. Tis is fine